src/chainstate/burn/operations/mod.rs

```diff
@@ -313,17 +314,34 @@ pub struct DelegateStxOp {
     pub burn_header_hash: BurnchainHeaderHash, // hash of the burn chain block header
 }
 
+fn hex_serialize<S: serde::Serializer>(bhh: &BurnchainHeaderHash, s: S) -> Result<S::Ok, S::Error> {
+    let inst = bhh.to_hex();
+    s.serialize_str(inst.as_str())
+}
+
+fn hex_deserialize<'de, D: serde::Deserializer<'de>>(
+    d: D,
+) -> Result<BurnchainHeaderHash, D::Error> {
+    let inst_str = String::deserialize(d)?;
+    BurnchainHeaderHash::from_hex(&inst_str).map_err(serde::de::Error::custom)
+}
+
```

Comment:
What's the reason for shuffling around these functions? I'd prefer to keep all the burnchain op structs declared together uninterrupted by serialization logic.

Reply:
What do you mean by shuffling? Where should these functions live instead?

Reply:
Oh, I read the diff wrong. My apologies! In this case, I simply would ask to move them to after the `PegInOp` declaration instead of before.

Reply:
Moved in 92ec39e4d36420fb9b5c966ac549a84ffc3417cc

Reply:
Thanks! :pray: